src/components/Text/Text.mdx

```diff
@@ -41,6 +41,9 @@ You may pass any Chakra `Box` prop, as well as the props below.
 The `Text` component renders paragraph `p` HTML elements which are inherently
 accessible and will be read by screen readers.
 
+You may pass an HTML `role` attribute if you need to explicitly set the ARIA
```

Comment:
It'd be helpful to elaborate or give examples on how and why you'd need to explicitly set a role for this component.

Reply:
Can be a new PR.

Reply:
I did a little research and I'm not sure a scenario in which you would want to add a `role` to a `p` tag (which is what `Text` ultimately boils down to). We had `role` as an explicit prop previously. Do you know why it was there?